test/index.test.js

```diff
@@ -1,41 +1,218 @@
 'use strict';
 
-const expect = require('chai').expect;
-const Hapi   = require('hapi');
+const Bluebird = require('bluebird');
+const expect   = require('chai').expect;
+const Hapi     = require('hapi');
+const Redis    = require('then-redis');
+
+const redisClient = Redis.createClient({
+  port: '6379',
+  host: 'localhost',
+  enable_offline_queue: false
```

Comment:
if redis server is down, it doesnt make sense to queue up request rates (we can't do anything about it anyway)